torch/_functorch/_aot_autograd/collect_metadata_analysis.py

```diff
@@ -301,6 +307,11 @@ def inner(*flat_args):
             if isinstance(o, torch.Tensor):
                 curr_storage = StorageWeakRef(o.untyped_storage())
                 out_tensor_alias_counts[curr_storage] += 1
+
+                if is_export and isinstance(o, FunctionalTensor):
+                    o_group_id = mode._partial_frozen_storage.get(curr_storage, None)
+                    if o_group_id in mode._mutated_partial_frozen_storage:
```

Comment:
nit: I'd write a helper method in `FunctionalTensorMode` that accepts a storage and returns if it was "frozen but also mutated", instead of poking at the private fields from AOTAutograd